fandogh_cli/namespace_commands.py

```diff
@@ -1,5 +1,5 @@
 import click
-from fandogh_cli.config import get_user_config
+from fandogh_cli.config import get_user_config, set_cluster_namespace
 
 from fandogh_cli.fandogh_client.namespace_client import *
 from .base_commands import FandoghCommand
```

Comment:
namespace list command does not show activated namespace after changing cluster.